website/docs/docs/dbt-cloud-apis/admin-cloud-api.md

```diff
@@ -3,11 +3,27 @@ title: "dbt Cloud Administrative API"
 id: "admin-cloud-api"
 ---
 
-The dbt Cloud Administrative API is enabled by default for _Team_ and _Enterprise_ plans. It can be used to:
+The dbt Cloud Administrative API is enabled by default for [_Team_ and _Enterprise_ plans](https://www.getdbt.com/pricing/). It can be used to:
```

Comment:
```suggestion
The dbt Cloud Administrative API is enabled by default for [Team and Enterprise plans](https://www.getdbt.com/pricing/). It can be used to:
```